syntax = "proto3";

message Result {
    bool isSuccess = 1;
    string message = 2;
}